x-pack/test/functional/apps/ml/anomaly_detection_jobs/categorization_job.ts

```diff
@@ -7,6 +7,7 @@
 
 import { CATEGORY_EXAMPLES_VALIDATION_STATUS } from '@kbn/ml-plugin/common/constants/categorization_job';
 import { FtrProviderContext } from '../../../ftr_provider_context';
+import { FieldStatsType } from '../data_frame_analytics/types';
```

Comment:
Might be worth to move this one level up from `data_frame_analytics` to a shared place in `ml`.

Reply:
Updated here [`562605c` (#151336)](https://github.com/elastic/kibana/pull/151336/commits/562605c7389ea551af023606428b9611f5016159)